README.md

```diff
@@ -37,26 +37,33 @@ e.g. the configuration settings for iterm2
 
 ## 🆙 append the powerlevel zsh configuration to your own .zshrc file
 
-you may want to customize the zsh configuration before appending it to your own configuration file.
-our  platform developers might find the customization for aws vault useful though. other than that the configuration might inspire you for
-your own hacks. let us know what you come up with.
+this repo contains a `zshrc` file which contains preconfigured, helpful configuration, check it out.
```

Comment:
this repo contains a `zshrc` file which contains preconfigured configuration defaults, that we ourselves find useful. check it out!